### ISSUE TITLE
Tutorial Part 8 "Introduction to Plans" missing required positional arg "plan_blueprint"

### ISSUE THREAD

**Issue by IanQS** [OpenMined/PySyft#2939]
**Describe the bug**
While trying to run the tutorial I get the error above. 

Running

```
@sy.func2plan()
def plan_double_abs(x):
    x = x + x
    x = torch.abs(x)
    return x
```

I get 

```
TypeError                                 Traceback (most recent call last)
<ipython-input-9-5cb858851dff> in <module>
----> 1 @sy.func2plan()
      2 def plan_double_abs(x):
      3     x = x + x
      4     x = torch.abs(x)
      5     return x

TypeError: func2plan() missing 1 required positional argument: 'plan_blueprint'
```

**To Reproduce**

Use Docker container
1) `docker container run openmined/pysyft-notebook`
2) open the link
3) Copy the github repo (use HTTPS)
4) Try and run `Part 08 - Introduction to Plans`

**Expected behavior**

No error

**Desktop (please complete the following information):**
 - OS: Manjaro Linux

`docker version` gives me 

```
Client:                                                                                                                                                                                                                                [1/327]
 Version:           19.03.5-ce                                                                                                                                                                                                                
 API version:       1.40
 Go version:        go1.13.4
 Git commit:        633a0ea838
 Built:             Fri Nov 15 03:19:09 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          19.03.5-ce
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.4
  Git commit:       633a0ea838
  Built:            Fri Nov 15 03:17:51 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.3.2.m
  GitCommit:        d50db0a42053864a270f648048f9a8b4f24eced3.m
 runc:
  Version:          1.0.0-rc9
  GitCommit:        d736ef14f0288d6993a1845745d6756cfc9ddd5a
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

Running `!pip list` in the notebook gives me 

```
Package              Version 
-------------------- --------
absl-py              0.7.1   
asn1crypto           0.24.0  
astor                0.8.0   
attrs                19.1.0  
backcall             0.1.0   
bleach               3.1.0   
certifi              2019.3.9
cffi                 1.12.2  
chardet              3.0.4   
Click                7.0     
conda                4.6.14  
cryptography         2.6.1   
decorator            4.4.0   
defusedxml           0.6.0   
entrypoints          0.3     
Flask                1.0.3   
Flask-SocketIO       4.1.0   
gast                 0.2.2   
grpcio               1.21.1  
h5py                 2.9.0   
idna                 2.8     
ipykernel            5.1.1   
ipython              7.5.0   
ipython-genutils     0.2.0   
ipywidgets           7.4.2   
itsdangerous         1.1.0   
jedi                 0.13.3  
Jinja2               2.10.1  
joblib               0.13.2  
jsonschema           3.0.1   
jupyter              1.0.0   
jupyter-client       5.2.4   
jupyter-console      6.0.0   
jupyter-core         4.4.0   
Keras-Applications   1.0.8   
Keras-Preprocessing  1.1.0   
lz4                  2.1.10  
Markdown             3.1.1   
MarkupSafe           1.1.1   
mistune              0.8.4   
mock                 3.0.5   
msgpack              0.6.1   
nbconvert            5.5.0   
nbformat             4.4.0   
notebook             5.7.8   
numpy                1.16.4  
pandocfilters        1.4.2   
parso                0.4.0   
pexpect              4.7.0   
pickleshare          0.7.5   
Pillow               6.0.0   
pip                  19.0.3  
prometheus-client    0.6.0   
prompt-toolkit       2.0.9   
protobuf             3.8.0   
ptyprocess           0.6.0   
pycosat              0.6.3   
pycparser            2.19    
Pygments             2.4.2   
pyOpenSSL            19.0.0  
pyrsistent           0.14.11 
PySocks              1.6.8   
python-dateutil      2.8.0   
python-engineio      3.8.1   
python-socketio      4.1.0   
PyYAML               5.1.1   
pyzmq                18.0.0  
qtconsole            4.5.1   
requests             2.21.0  
ruamel-yaml          0.15.46 
scikit-learn         0.21.2  
scipy                1.3.0   
Send2Trash           1.5.0   
setuptools           41.0.0  
six                  1.12.0  
sklearn              0.0     
syft                 0.1.18  
tblib                1.4.0   
tensorboard          1.13.1  
tensorflow           1.13.1  
tensorflow-estimator 1.13.0  
termcolor            1.1.0   
terminado            0.8.2   
testpath             0.4.2   
tf-encrypted         0.5.5   
torch                1.1.0   
torchvision          0.3.0   
tornado              6.0.2   
traitlets            4.3.2   
urllib3              1.24.1  
wcwidth              0.1.7   
webencodings         0.5.1   
websocket-client     0.56.0  
websockets           7.0     
Werkzeug             0.15.4  
wheel                0.33.1  
widgetsnbextension   3.4.2   
zstd                 1.4.0.0 
```


**Additional context**

**Comment by IanQS** [comment]
Can't recreate this on a different machine that has`syft 0.2.0a2` so I guess the docker version is behind? 

I didn't install the native version on this machine (the one with the problem) because of #2931

**Comment by karlhigley** [comment]
Yeah, syft 0.1.18 is at least six months old, and there have been fixes to Plans and blueprints since then.

**Comment by LaRiffle** [comment]
@IanQS is this issue solved?

**Comment by IanQS** [comment]
> @IanQS is this issue solved?

@LaRiffle  Apologies, I haven't followed up since #2931 got solved. I can do it later today and try to open a docker image PR

**Comment by IanQS** [comment]
@LaRiffle 

Yeap, still an issue. I pulled a fresh image and I'm still seeing the same problem. If I had to guess, it's because the docker image is on 

`syft                 0.1.18`

and the anaconda version is on 

`syft                   0.2.2a1`

but it might be a lack of 

`syft-proto             0.1.0a1.post36` on the docker image too. Is there a fix already OTW? I can try but I'm not very proficient with Docker
